package-lock.json

```diff
@@ -0,0 +1,4650 @@
+{
```

Comment:
there is already a `yarn.lock` file in this directory

remove the newly added `package-lock.json` file